src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -568,14 +569,14 @@ public void StopStake()
                 if (hashBlock == null) continue;
 
                 var utxoStakeDescription = new UtxoStakeDescription
-                           {
-                               TxOut = utxo,
-                               OutPoint = new OutPoint(coinSet.TransactionId, outputReference.Transaction.Index),
-                               Address = outputReference.Address,
-                               HashBlock = hashBlock,
-                               UtxoSet = coinSet,
-                               Secret = walletSecret // Temporary.
-                           };
+                   {
```

Comment:
Still not the right code style as is used elsewhere

Reply:
hopefully not a blocker!